packages/core/src/QueryManager.js

```diff
@@ -49,8 +84,9 @@ export class QueryManager {
       if (cache) {
         const cached = this.clientCache.get(sql);
         if (cached) {
+          const data = await cached;
           this._logger.debug('Cache');
-          result.fulfill(cached);
+          result.prepare(type === 'exec' ? null : data);
```

Comment:
It's a separate issue, but this makes me think we should double check caching interactions with exec queries. I don't think we want to be going to the cache for any exec queries...

Reply:
It looks like [cache is set to false by default](https://github.com/uwdata/mosaic/blob/76e4874f05eacb27452e85f49c38d5adddd3dfbf/packages/core/src/QueryManager.js#L40) and `exec` [makes a query without cache](https://github.com/uwdata/mosaic/blob/76e4874f05eacb27452e85f49c38d5adddd3dfbf/packages/core/src/Coordinator.js#L118) so it should always be off.

Reply:
Good, sounds like younger me did think it through :)